app/custom_domain_validation.py

```diff
@@ -39,9 +40,9 @@ def validate_dkim_records(self, custom_domain: CustomDomain) -> dict[str, str]:
         Returns empty list if all records are ok. Other-wise return the records that aren't properly configured
         """
         invalid_records = {}
-        for prefix, expected_record in self.get_dkim_records():
+        for prefix, expected_record in self.get_dkim_records().items():
```

Comment:
Did it fail before?

Reply:
No, but it was returning a list of tuples and using it as a dict, so now it's changed to a dict